docs/build/build-hrmp-channels.md

```diff
@@ -7,104 +7,72 @@ keywords: [HRMP, parachain, statemint, proposal, asset hub, statemine]
 slug: ../build-hrmp-channels
 ---
 
-[HRMP](../learn/learn-xcm.md#xcmp-lite-hrmp) has the same interface and functionality as
-[XCMP](../learn/learn-xcm.md#xcmp-cross-chain-message-passing) but is much more demanding on
-resources since it passes all messages via the Relay Chain. When XCMP is implemented on Polkadot,
-HRMP is planned to be deprecated and phased out in favor of it.
+In order to communicate over [HRMP](../learn/learn-xcm-transport.md#hrmp-xcmp-lite), parachains must
+establish channels by registering them on the Relay Chain. Like
+[XCMP](../learn/learn-xcm-transport.md#xcmp-cross-chain-message-passing), HRMP is a message
+transport protocol, but passes all messages via the Relay Chain. When XCMP is implemented on
+Polkadot, HRMP is planned to be deprecated and phased out.
 
-## Opening HRMP channel: parachain to parachain
+HRMP channels are uni-directional. Bi-directional communication between two parachains will require
+two channels, one in each direction.
 
-To open a channel from one parachain to another that are not system chains on Polkadot, the
-governance of each parachain needs to declare its intent to open a channel on the Relay Chain, and
-the second chain needs to accept and do the same.
+## Opening HRMP Channels
 
-In order to do this, the calls to be dispatched from both chains are:
+Opening a channel between two parachains is a two-phase process, with one chain first initiating a
+channel request and then the second chain accepting it. When neither chain is a system chain, they
+will use the `hrmpInitOpenChannel` and `hrmpAcceptOpenChannel` calls, respectively.
 
-1. `hrmp > hrmpInitOpenChannel(recipient, proposedMaxCapacity, proposedMaxMessageSize)`: this call
-   initiates opening a channel from a parachain to a given recipient with given channel
+Each chain must dispatch the following calls on the Relay Chain from its parachain origin.
 
-2. `hrmp > hrmpAcceptOpenChannel(sender)`: accepting the channel open request from the given sender.
+1. `hrmp > hrmpInitOpenChannel(recipient, proposedMaxCapacity, proposedMaxMessageSize)`: Initiates
+   channel establishment by creating a channel request with a given configuration. Note that the max
+   capacity and max message size must be within the `configuration`'s limits.
 
-Each parachain is to use its own governance process to execute this. The call will be executed on
-the Relay Chain.
+2. `hrmp > hrmpAcceptOpenChannel(sender)`: Accept the channel open request from the given sender.
+
+In order to dispatch a call from its sovereign origin, a parachain may use governance to send the
+encoded call in a `Transact` instruction to the Relay Chain, but it may also execute this logic
+autonomously (e.g. on the notification that a channel was requested).
 
```

Comment:
```suggestion

The xcm-executor pallet can be forked by parachains' teams and modified to include logic to handle [high-level HRMP instructions](https://paritytech.github.io/xcm-docs/journey/channels-and-bridges.html#hrmpnewchannelopenrequest). These instructions are not included in the default version of the pallet, but can be used for opening, accepting, and closing channels without relying on governance.

```

Reply:
Let us add this info as a separate PR, after merging this PR's content that has streamlined information.